src/router.js

```diff
@@ -21,6 +21,14 @@ export default new Router({
       // which is lazy-loaded when the route is visited.
       component: () => import(/* webpackChunkName: "about" */ './views/About.vue'),
     },
+    {
+      path: '/editprofile',
+      name: 'editprofile',
+      // route level code-splitting
+      // this generates a separate chunk (about.[hash].js) for this route
+      // which is lazy-loaded when the route is visited.
+      component: () => import(/* webpackChunkName: "about" */ './views/EditProfile.vue'),
```

Comment:
```suggestion
      component: () => import(/* webpackChunkName: "edit-profile" */ './views/EditProfile.vue'),
```
これは罠なんですが、`/* webpackChunkName: "about" */`は実はwebpack用の特別なコメントで、このままだとwebpackがEditProfile.vueをAbout.vueと同じchunkに入れちゃうので、chunkNameを変えておくといい感じにファイルが分割されていいと思います。

Reply:
コード移してたときに見逃してました。明日中に修正してあげます